lib/generate-report.js

```diff
@@ -183,7 +183,7 @@ function generateReport(options) {
 			feature.isPending = false;
 			suite.featureCount.total++;
 			var idPrefix = staticFilePath ? '' : `${ uuid() }.` ;
-			feature.id = `${ idPrefix }${ feature.id }`.replace(/[^a-zA-Z0-9-_]/g, '-');
+			feature.id = `${ idPrefix }${ feature.id }`.split(" ").join("-");
```

Comment:
This fix is not correct, see also my comment here https://github.com/wswebcreation/multiple-cucumber-html-reporter/pull/148. 
Maybe you can remove it from your PR